packages/qwik/src/server/v2-ssr-container.ts

```diff
@@ -956,7 +956,7 @@ class SSRContainer extends _SharedContainer implements ISSRContainer {
     let skipCount = elementIdx - lastSerializedIdx;
     // console.log('emitVNodeSeparators', lastSerializedIdx, elementIdx, skipCount);
     while (skipCount != 0) {
-      if (skipCount >= 4096) {
```

Comment:
@mhevery I thing this is a good change, right? If skipCount is equal to 4096 then we want to execute code in the else block, right?

Reply:
yes, that is correct. Thank you for noticing.